### PR TITLE
Fixed #6727 - patch_cache_control now properly patches an empty string

### PR DESCRIPTION
**Pull request by dwightgunning**

### CODE REVIEW

File: django/utils/cache.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def dictvalue(t):
         else:
             return '%s=%s' % (t[0], t[1])
 
-    if response.has_header('Cache-Control'):
+    if response.has_header('Cache-Control') and response['Cache-Control']:
```
Comment (charettes) [inline django/utils/cache.py]: `response.get('Cache-Control')` should do.

Comment (dwightgunning) [inline django/utils/cache.py]: Good point. I've made the change.

**Comment by charettes** [comment]
Buildbot, test this please.

**Comment by timgraham** [comment]
merged in 1f29164ced01bde92c72d20d67bc453484a4f078, thanks!

**Pull request closed without merging**